contents/devops/DevOps.md

```diff
@@ -0,0 +1,89 @@
+# DevOps
+
+Authors: [John Yong](https://github.com/whipermr5)
+
+## What is DevOps?
+
+> &ldquo;DevOps is the practice of operations and development engineers participating together in the entire service lifecycle, from design through the development process to production support.&rdquo; &mdash; [the agile admin](https://theagileadmin.com/what-is-devops/)
+
+The term **DevOps** comes from the amalgamation of two words - **Dev**elopment and **Op**eration**s**. While there are many definitions and opinions out there about what DevOps really is, and no one true answer, most agree that it is the idea of Developers and Operations working together to achieve a goal ([source](https://blog.xebialabs.com/2015/04/23/best-guide-to-getting-started-in-devops/)). This involves a **shift in mindset** away from the traditional view of Development and Operations as two distinct teams with few overlapping responsibilities, to a new mindset where developers and operations engineers communicate, collaborate and share the responsibility of developing and delivering software.
+
+Indeed, some see DevOps as a culture shift that involves more than just developers and operations teams, but also everyone involved in delivering value to end users, including testers and Quality Assurance, Project Management and Human Resources teams ([source](http://blog.nwcadence.com/devops-culture-not-team/)). However, this document will focus more on the development and operations side of DevOps, which is what budding software engineers like yourself need to know.
+
+> Tip: for a comprehensive discussion of what DevOps is and is not, refer to *the agile admin*'s article, [What Is DevOps?](https://theagileadmin.com/what-is-devops/)
+
+## Aspects of DevOps
+
+While DevOps is predominantly an organisational culture and mindset, there are several aspects it covers that you, as a software engineering student, need to be familiar with before entering the industry. These are outlined in the DevOps toolchain ([source](https://en.wikipedia.org/wiki/DevOps#DevOps_toolchain)):
+
+1. Code
+2. Build
+3. Test
+4. Package
+5. Release
+6. Configure
+7. Monitor
+
+These can be roughly summarised into four major aspects:
+
+### Code Development & Review
+
+This includes using version control tools, following a process when developing, frequently merging code with a central branch, and going through a code review process.
+
+#### Relevant Tools
+
+- Git
+- GitHub
```

Comment:
May be you can mention these as examples. There are other vialble alternatives to these.